spec/features/access_data_flow_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+
+RSpec.feature 'View register', type: :feature do
+  before(:all) do
+    country_data = File.read('./spec/support/country.rsf')
```

Comment:
I haven't thought about this in detail, but do you think we could improve some of this set up and stubbing in the feature tests?

It seems like some of this setup is common to multiple tests, and if the tests aren't actually testing the consumer process that reads data in from ORJ there would be easier ways to get to a state where we have a few registers in the DB and then you can just focus on testing the user journey.

Reply:
I agree. I think this is too big for this PR, so I'll raise a ticket that we need to look at the testing set up  (and rolled into this, also the current difficulty doing any browser-based JavaScript testing.)